integrations/segmentio/lib/index.js

```diff
@@ -315,10 +317,31 @@ Segment.prototype.normalize = function(message) {
   }
   if (this.options.addBundledMetadata) {
     var bundled = keys(this.analytics.Integrations);
+    var maybeBundledConfigIds = this.options.maybeBundledConfigIds
+
+    // Generate a list of bundled config IDs using the intersection of
+    // bundled destination names and maybe bundled config IDs.
+    var bundledConfigIds = []
+    for (var i = 0; i < bundled.length; i++) {
+      var name = bundled[i]
+      if (!maybeBundledConfigIds) {
+        break
+      }
+      if (!maybeBundledConfigIds[name]) {
+        continue
+      }
+
+      for (var j = 0; j < maybeBundledConfigIds[name].length; j++) {
+        var id = maybeBundledConfigIds[name][j]
+        bundledConfigIds.push(id)
+      }
+    }
+
+
     msg._metadata = msg._metadata || {};
     msg._metadata.bundled = bundled;
+    msg._metadata.bundledConfigIds = bundledConfigIds;
     msg._metadata.unbundled = this.options.unbundledIntegrations;
```

Comment:
```suggestion
	msg._metadata.unbundled = this.options.unbundledIntegrations;
    msg._metadata.bundledConfigIds = bundledConfigIds;
```

nit, I would keep order the same in case a system expects consistent json key order.

Reply:
In my stage tests, the order doesn't affect behavior.